transport-classes-io_uring/src/main/java/io/netty/channel/uring/AbstractIoUringServerChannel.java

```diff
@@ -184,6 +223,15 @@ protected void readComplete0(byte op, int res, int flags, short data, int outsta
 
             if (res >= 0) {
                 allocHandle.incMessagesRead(1);
+                final long acceptedAddressMemoryAddress;
+                final long acceptedAddressLengthMemoryAddress;
+                if (acceptedAddressMemory == null) {
```

Comment:
Perhaps it would be better to use IoUring.isIOUringAcceptMultishotSupported(), as it is a constant.

Reply:
Changed it...